docs/src/docs/asciidoc/user/clustered-cache.adoc

```diff
@@ -143,6 +143,11 @@ In this example, a dedicated pool of 32MB is allocated for clustered-cache from
 <7> Configures another cache (`shared-cache-2`) that shares the resource pool (`resource-pool-a`) with `shared-cache-1`.
 <8> Creates fully initialized cache manager with the clustered caches.
 
+NOTE:  When multiple caches are sharing a shared pool, eviction operation called by a cache evicts elements of that cache
+itself and not just any set of elements in the shared pool. This means that today, when space in a shared pool
+is allocated to a cache, it remains with it for its use only. Improvisation would be made in future to allow eviction of
```

Comment:
Improvisation?!?

Reply:
And we should not talk about the future. This is the current documentation

Reply:
removed